### PR TITLE
EIA: Add preliminary schema-less support for Petroleum and Natural Gas datasets

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

File: scripts/us_eia/opendata/process/test_data/ng.mcf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Node: dcid:eia/ng.n3035_4.m
```
Comment (pradh) [inline scripts/us_eia/opendata/process/test_data/ng.mcf:1]: Do we want to keep this in its original capital case?

File: scripts/us_eia/opendata/process/common.py [hunk 2]
```diff
@@ -53,27 +54,41 @@ def _parse_date(d):
...
+
     return None
 
 
 def _eia_dcid(raw_sv):
     return 'dcid:eia/' + raw_sv.lower()
 
 
+def _enumify(in_str):
+    return in_str.title().replace(' ', '')
```
Comment (beets) [inline scripts/us_eia/opendata/process/common.py:72]: do we need to ensure camel case as well?

File: scripts/us_eia/opendata/process/ng.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+"""EIA Natural Gas Dataset specific functions."""
+
+import re
+
+# List of series in Pattern #1 that include non-US countries. Unfotunately this
```
Comment (beets) [inline scripts/us_eia/opendata/process/ng.py:5]:
```suggestion
# List of series in Pattern #1 that include non-US countries. Unfortunately this
```

Comment (beets) [reply scripts/us_eia/opendata/process/ng.py:5]: also, could you describe the significance of the N/S prefix?

File: scripts/us_eia/opendata/process/test_data/ng.mcf (hunk 1 above)
Comment (beets) [reply scripts/us_eia/opendata/process/test_data/ng.mcf:1]: yes, i think that would look more familiar and might be easier for exploration in the near term (e.g. if we want to look up a particular series in the browser)

**Review by beets — APPROVED** [review]
nice!

File: scripts/us_eia/opendata/process/common.py (hunk 2 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/common.py:72]: title() approx does that?

https://stackoverflow.com/questions/3728655/titlecasing-a-string-with-exceptions

File: scripts/us_eia/opendata/process/test_data/ng.mcf (hunk 1 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/test_data/ng.mcf]: SG.  Stuck to original case

File: scripts/us_eia/opendata/process/ng.py (hunk 3 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/ng.py]: Done.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

File: scripts/us_eia/opendata/process/common.py (hunk 2 above)
Comment (beets) [reply scripts/us_eia/opendata/process/common.py:72]: oh, i missed that there was a call to "title" -- sg

**Pull request merged**